.evergreen/config.yml

```diff
@@ -1412,7 +1408,7 @@ buildvariants:
       ocsp-algorithm: ecdsa
       ocsp-must-staple: on
       ocsp-delegate: on
-      ruby: [ruby-3.0, ruby-2.7, ruby-2.3]
```

Comment:
I'm pretty sure 2.3 is here as the lower bound of the supported versions, so I think it needs to be changed to 2.4 rather than removed unless ocsp+2.4 is tested elsewhere.

Reply:
I see that 2.3 had special code, so argument can be made that 2.4 configuration should've already been in place in addition to 2.3. I think we should test 2.4 as the lower bound.